test/login.test.ts

```diff
@@ -0,0 +1,37 @@
+import { chromium, Page, Browser, BrowserContext } from "playwright"
+
+describe("login", () => {
+  let browser: Browser
+  let page: Page
+  let context: BrowserContext
+
+  beforeAll(async () => {
+    browser = await chromium.launch()
+    context = await browser.newContext()
+  })
+
+  afterAll(async () => {
+    await browser.close()
+  })
+
+  beforeEach(async () => {
+    page = await context.newPage()
+  })
+
+  afterEach(async () => {
+    await page.close()
+    // Remove password from local storage
+    await context.clearCookies()
+  })
+
+  it("should be able to login with the password from config.yml", async () => {
```

Comment:
Is this a WIP? There's nothing being read from `config.yml`.

Reply:
yes, sorry need to update! it was working but only intermittently